### PR TITLE
Registry auth + image build / push

### PR DESCRIPTION
**Pull request by WillBeebe**

### CODE REVIEW

File: cmd/containers.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+func init() {
+	rootCmd.AddCommand(imageCmd)
+
+	// imageCmd.AddCommand(imageBuildCmd)
+	// imageCmd.AddCommand(imageListCmd)
+	// imageCmd.AddCommand(imagePushCmd)
+	imageCmd.AddCommand(packageImgCmd)
+}
+
+// func runImageBuild(cmd *cobra.Command, args []string) error {
```
Comment (WillBeebe) [inline cmd/containers.go:55]: These are here so it's easy to try different UX flows. Do we expose build, do we combine build and push, etc... What feels good right now? Then, I'll delete the rest.

File: pkg/containers/utils.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+// TOKEN=$(aws ecr get-authorization-token --output text --query 'authorizationData[].authorizationToken')
+// curl -i -H "Authorization: Basic $TOKEN" https://aws_account_id.dkr.ecr.region.amazonaws.com/v2/amazonlinux/tags/list
+// Azure
+// https://learn.microsoft.com/en-us/azure/container-registry/container-registry-authentication?tabs=azure-cli
+// GCP
+// Pushing an Image
+// make install.macos && \
+//   REGISTRY_AUTH_TOKEN=$(gcloud auth print-access-token --impersonate-service-account massdriver-sa@md-wbeebe-0808-example-apps.iam.gserviceaccount.com) && \
+//   mass image push
+func getAuthConfig(authToken string, image string) (string, error) {
```
Comment (WillBeebe) [inline pkg/containers/utils.go:38]: most useful part

**Comment by coryodaniel** [comment]
we can close this now, yeah?

**Comment by WillBeebe** [comment]
Yes, absolutely

**Pull request closed without merging**